exporter/cat1_view_helper.go

```diff
@@ -15,29 +15,17 @@ import (
 	"github.com/projectcypress/cdatools/models"
 )
 
-func escape(i interface{}) string {
-	switch str := i.(type) {
-	case string:
-		return escapeString(str)
-	case int64:
-		return escapeString(strconv.FormatInt(str, 10))
-	case int:
-		return escapeString(strconv.Itoa(str))
-	case *int64:
-		if str != nil {
-			return escapeString(strconv.FormatInt(*str, 10))
-		}
-	case *int:
-		if str != nil {
-			return escapeString(strconv.Itoa(*str))
-		}
-	}
-	return ""
+// i64ptrToString converts the value of an *int64 to a string.
+func i64ptrToString(i *int64) string {
```

Comment:
I struggled with the naming. I thought about adopting a "c-ism" and saying `i64toa`, but it accepts pointers to int64, so that wasn't right and `toa` never made sense to me when I was first exposed to it until I looked it up so I figured `ToString` was worth the extra characters. So I decided to take a different "c-ism" which is the tendency to append `ptr` to a data type name. That's why the `ptr` is not capitalized is because it's technically a part of the "word" `i64ptr`. So those are the thoughts that went on in my head, I'm open to suggestions as it seems like the function isn't actually used as much as I thought so length seems to not matter as much as I thought it would.